File: src/addon365.Web.API/ClientApp/src/app/models/business-contact.ts
import { AddressMaster } from "./address-master";
import { Contact } from "./contact";

export class BusinessContact {
  businessName: string;
  contactAddress: AddressMaster;
  landline: string;
  mobileNumber: string;
  secondaryMobileNumber: string;
  proprietor:Contact;
  contactPerson:Contact;
}

File: src/DotNet/Should be removed/addon365.Erp.DataEntity/Crm/BusinessContact.cs
﻿namespace addon365.Database.Entity.Crm
{
    public class BusinessContact : BaseEntityWithLogFields
    {
        public string BusinessName { get; set; }
        public string BusinessMailId { get; set; }
        public Address.Master ContactAddress { get; set; }
        public Contact Proprietor { get; set; }
        public Contact ContactPerson { get; set; }
        public string Landline { get; set; }
        public string MobileNumber { get; set; }
        public string SecondaryMobileNo { get; set; }
    }
}
